### PR TITLE
mimic: client: flush cap release on session flush

### PR DESCRIPTION
**Pull request by pdvian**
http://tracker.ceph.com/issues/38103

### CODE REVIEW

File: src/client/Client.cc [hunk 1]
```diff
@@ -2119,6 +2119,10 @@ void Client::handle_client_session(MClientSession *m)
     break;
 
   case CEPH_SESSION_FLUSHMSG:
+    /* flush cap release */
+    if (auto& m = session->release; m) {
+      session->con->send_message(std::move(m));
```
Comment (batrick) [inline src/client/Client.cc]: `session->con->send_message(m.detach());`

**Comment by smithfarm** [comment]
@pdvian Please leave a comment here once you have made the change requested by @batrick - thanks!

**Comment by smithfarm** [comment]
closing in favor of https://github.com/ceph/ceph/pull/26424
